website/content/docs/concepts/acl/auth-methods/oidc.mdx

```diff
@@ -110,8 +217,19 @@ port numbers, and whether trailing slashes are present.
   what you expect. Since claims data is logged verbatim and may contain
   sensitive information, do not use this option in production.
 
+- For client assertions, if `VerboseLogging` is enabled, then the Nomad leader
+  server logs a JWT when the auth method is created, and when someone makes
+  a login attempt. These JWTs are not 100% identical to what gets sent to the
+  OIDC provider due to being time-bounded, but you can check the JWT headers
```

Comment:
```suggestion
  OIDC provider due to being time-bound, but you can check the JWT headers
```
I doubled-checked this in a dictionary

Reply:
I woulda sworn I changed that! ... oh yes, I did, in 2 other places, but not also this 3rd place :upside_down_face: